assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -1,59 +1,51 @@
 import { IMAGE_SIZES } from './imageSizes';
-const { __ } = wp.i18n;
 
-export const CampaignCovers = ({ covers, initialRowsLimit, row, loadMoreCovers, inEditor = false }) => {
-  const amountPerRow = 3;
-  const showLoadMore = !!initialRowsLimit && covers.length > amountPerRow * row;
+const { __ } = wp.i18n;
 
-  return (
-    <>
-      <div className='thumbnail-largeview-container'>
-        {covers.map((cover, index) => {
-          const { href, image, alt_text, name, src_set } = cover;
-          const hideCover = !!initialRowsLimit && index >= row * amountPerRow;
+export const CampaignCovers = ({
+  covers,
+  initialRowsLimit,
+  row,
+  inEditor = false,
+  isCarouselLayout,
+  amountOfCoversPerRow,
+}) => (
+  <div className='covers'>
+    {covers.map((cover, index) => {
+      const { href, image, alt_text, name, src_set } = cover;
+      const hideCover = !isCarouselLayout && !!initialRowsLimit && index >= row * amountOfCoversPerRow;
 
-          if (hideCover) {
-            return null;
-          }
+      if (hideCover) {
+        return null;
+      }
 
-          const campaignLink = inEditor ? null : href;
+      const campaignLink = inEditor ? null : href;
 
-          return (
-            <div key={name} className='campaign-card-column'>
-              <a
-                href={campaignLink}
-                data-ga-category='Campaign Covers'
-                data-ga-action='Image'
-                data-ga-label='n/a'
-                aria-label={__('Check our campaign about ' + name, 'planet4-blocks')}
-              >
-                <div className='thumbnail-large'>
-                  {image && image[0] &&
-                    <img
-                      loading='lazy'
-                      sizes={IMAGE_SIZES.campaign}
-                      srcSet={src_set}
-                      src={image[0]}
-                      alt={alt_text}
-                      title={alt_text}
-                    />
-                  }
-                  <span className='yellow-cta'><span aria-label='hashtag'>#</span>{name}</span>
-                </div>
-              </a>
+      return (
+        <div key={name} className='cover'>
+          <a
+            href={campaignLink}
+            data-ga-category='Campaign Covers'
+            data-ga-action='Image'
+            data-ga-label='n/a'
+            aria-label={__('Check our campaign about ' + name, 'planet4-blocks')}
+          >
+            <div className='thumbnail-large'>
+              {image && image[0] &&
```

Comment:
Why do directly check for length `{image.length && ...`?

Reply:
This was already [written like this](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/blocks/Covers/CampaignCovers.js#L31), so I'm not sure what the initial logic behind it was but it is shorter anyway to write it like this than it is to write `image && image.length > 0` so I'm ok with it 🤷‍♀️